build.gradle

```diff
@@ -313,6 +313,12 @@ project(':core') {
     from sourceSets.test.output
   }
 
+  task publishDocs(type: Jar) {
```

Comment:
Gwen, you may be right about the tarball and site, but javadoc artifacts are a standard Maven thing (along with jars and sources jars), eg see here:

http://repo1.maven.org/maven2/org/scala-lang/scala-library/2.11.7/

IDEs and other tools can use those jars.

@omkreddy, will the docs classifier in Gradle generate a jar with the javadoc suffix in the name?